src/pkg/packager/remove.go

```diff
@@ -98,10 +80,35 @@ func (p *Packager) Remove(packageName string) error {
 
 					return err
 				}
+
+				// Remove the uninstalled chart from the list of installed charts
+				deployedPackage.DeployedComponents[i].InstalledCharts = deployedPackage.DeployedComponents[i].InstalledCharts[:h]
```

Comment:
This is better because it is possible to interrupt the uninstall process (i.e. timeout, ctrl-c or otherwise) leaving a component partially uninstalled.  Doing this allows for Zarf to resume where it left off more often.